engine/render/src/render/render.cpp

```diff
@@ -294,7 +294,7 @@ namespace dmRender
         {
             if (!context->m_OutOfResources)
             {
-                dmLogWarning("Renderer is out of resources, some objects will not be rendered.");
+                dmLogWarning("Max number of draw calls reached (%u), some objects will not be rendered. Increase the capacity with graphics.max_draw_calls", context->m_RenderObjects.Capacity());
```

Comment:
Something I need to look into is why we can runout of render objects, while the profiler doesn't report a number close to the max_draw_calls.

My guess is that we register render objects that will later be skipped (e.g. 0 vertices)